core/src/main/java/io/jenkins/pluginhealth/scoring/scores/Scoring.java

```diff
@@ -27,17 +27,47 @@
 import io.jenkins.pluginhealth.scoring.model.Plugin;
 import io.jenkins.pluginhealth.scoring.model.ScoreResult;
 
+/**
+ * Represents a scoring process of a plugin, based on ProbeResults contained within the Plugin#details map.
+ */
 public abstract class Scoring {
+    /**
+     * Starts the scoring process of the plugin.
+     *
+     * @param plugin the plugin to score
+     * @return a {@link ScoreResult} describing the plugin based on the ProbeResult and the scoring requirements.
+     */
     public final ScoreResult apply(Plugin plugin) {
         return doApply(plugin);
     }
 
+    /**
+     *
```

Comment:
```suggestion
     * Applies the scoring implementation on a plugin.
     * Based on the provided plugin, the method returns a non-null {@link ScoreResult}.
     *
```
Missing Description?